plexpy/datafactory.py

```diff
@@ -1181,10 +1194,10 @@ def get_watch_time_stats(self, rating_key=None, grouping=None, query_days=None):
                                 'COUNT(DISTINCT %s) AS total_plays, section_id ' \
                                 'FROM session_history ' \
                                 'JOIN session_history_metadata ON session_history_metadata.id = session_history.id ' \
-                                'WHERE (session_history.grandparent_rating_key = ? ' \
-                                'OR session_history.parent_rating_key = ? ' \
-                                'OR session_history.rating_key = ?)' % group_by
-                        result = monitor_db.select(query, args=[rating_key, rating_key, rating_key])
+                                'WHERE (session_history.grandparent_rating_key IN %s ' \
+                                'OR session_history.parent_rating_key IN %s ' \
+                                'OR session_history.rating_key IN %s)' % (group_by, rating_keys, rating_keys, rating_keys)
+                        result = monitor_db.select(query)
```

Comment:
Just in case to protect against SQL injection.
```suggestion
                                'WHERE (session_history.grandparent_rating_key IN (%s) ' \
                                'OR session_history.parent_rating_key IN (%s) ' \
                                'OR session_history.rating_key IN (%s))' % (
                                    group_by, rating_keys_arg, rating_keys_arg, rating_keys_arg
                                )
                        result = monitor_db.select(query, args=[rating_keys, rating_keys, rating_keys])
```